Files/Commands/Paste.cs

```diff
@@ -202,10 +203,17 @@ await AppInstance.FilesystemViewModel.GetFolderFromPathAsync(destinationPath)
 
                     if (deleted == FilesystemErrorCode.ERROR_UNAUTHORIZED)
                     {
-                        // Try again with DeleteFileFromApp
-                        if (!NativeFileOperationsHelper.DeleteFileFromApp(item.Path))
+                        // Try again with fulltrust process
+                        if (AppInstance.FilesystemViewModel.Connection != null)
```

Comment:
Is there ever a scenario where we want to try again with the *FromApp method?

Reply:
Yes, if we want to delete the file permanently we can use deletefilefromapp.
Problem is it doesn't work for folders, the method for folders (removedirectory) only works for empty folders so I'd need to recursively delete the folder contents.
I was too lazy to do that and this is much simpler

Reply:
@gave92 Thanks for the explanation that should suffice for now.